### PR TITLE
Projectionizer: fix coding style

### PR DESCRIPTION
**Pull request by tristan0x**

### CODE REVIEW

File: bbp/nse/projectionizer/default.nix [hunk 1]
```diff
@@ -1,72 +1,69 @@
...
+    scipy
+    setuptools
+    tqdm
+    voxcell
+  ];
 
-    checkPhase = ''
-        nosetests
-    '';
+  passthru = {
```
Comment (wizmer) [inline bbp/nse/projectionizer/default.nix]: What does it do ?

**Comment by wizmer** [comment]
Do you use a linter for this ?

File: bbp/nse/projectionizer/default.nix (hunk 1 above)
Comment (tristan0x) [reply bbp/nse/projectionizer/default.nix]: `passthru` allows you to access package properties outside the derivation.
https://nixos.org/nixpkgs/manual/

Here it means that anywhere you can access package propagatedBuildInputs through `projectionizer.pythonDeps` property.

**Comment by tristan0x** [comment]
> Do you use a linter for this ?

I wish I did :)

**Pull request merged**